database/migrations/postgres/1722591376495565_indexes_deployments_environments.up.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX IF NOT EXISTS deployments_idx  ON deployments (appName, envname);
```

Comment:
please fix the typos in the commit message
"looging for, educing"
and "a new table tahat"